includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -688,28 +686,42 @@ public function scripts() {
 		}
 	}
 
+	/**
+	 * Returns true if the current page supports Payment Request Buttons, false otherwise.
+	 *
+	 * @since   5.3.0
+	 * @version 5.3.0
+	 * @return  boolean  True if the current page is supported, false otherwise.
+	 */
+	private function is_page_supported() {
+		return $this->is_product()
+			|| is_checkout()
+			|| is_cart()
+			|| WC_Stripe_Helper::has_cart_or_checkout_shortcode_on_current_page()
```

Comment:
After rebase this will be redundant because `has_cart_or_checkout_on_current_page` checks for `is_cart` and `is_checkout` now due to the latest change for the patch release.

Reply:
Nice catch! Fixed in 1f0eb7a7f342658d1efeb44a67fd13bd4f2a8d32